### ISSUE TITLE
Set Selectize.js maxOptions to match the field’s limit value

### ISSUE THREAD

**Issue by tmslnz** [hananils/association_ui_selector#48]
It’s a simple fix, not sure if it’s worth a PR or you can DYI :)
Happy to submit it as a PR if you prefer. Let me know.

I bumped into it with a 1800+ entry section. Thought it was caching, but turns out it’s Selectize’s default 1000 render limit.

This is probably best addressed with search-request-as-you-type, but for now this should do.

At: https://github.com/hananils/association_ui_selector/blob/master/assets/aui.selector.publish.js#L38

```js
// Only added `maxOptions` line
// maxOptions: (limit === 0) ? 1000 : limit,

storage.selectize({
	maxOptions: (limit === 0) ? 1000 : limit,
	preload: (limit === 0),
	sortField: [{
		field: 'text', 
		direction: 'asc'
	}],
	// … rest of options / callbacks
})
```

**Comment by nitriques** [comment]
> Happy to submit it as a PR if you prefer.

Yes please! It's easier to keep track of who modified what and for what reason! Thanks!

P.S. Pull requests also gives you karma points ;)
